configure.ac

```diff
@@ -0,0 +1,191 @@
+            AC_MSG_ERROR([sort not found])
+        fi
+        
+		;;
+	src)
+        AC_MSG_RESULT([stage1 will build systemd from source])
+
+        # these are needed for systemd build
+        AC_CHECK_PROG([INTLTOOLIZE],[intltoolize],[intltoolize])
+        if test x"${INTLTOOLIZE}" == x ; then
+            AC_MSG_ERROR([intltoolize not installed, apt-get install intltool])
+        fi
+
+        AC_CHECK_PROG([LIBTOOLIZE],[libtoolize],[libtoolize])
+        if test x"${LIBTOOLIZE}" == x ; then
+            AC_MSG_ERROR([libtoolize not installed, apt-get install libtool])
+        fi
+		;;
+	*)
+		AC_MSG_ERROR([stage1 build type '${RKT_STAGE1_USR_FROM}' unsupported])
+esac
+
+AC_SUBST(RKT_STAGE1_USR_FROM)
+
+## STAGE1: Systemd git path
+AC_ARG_WITH(stage1-systemd-src,
+        AS_HELP_STRING([--with-stage1-systemd-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_SYSTEMD_SRC="$withval"],
+        [RKT_STAGE1_SYSTEMD_SRC="git://anongit.freedesktop.org/systemd/systemd"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_SRC)
+
+## STAGE1: Systemd version
+AC_ARG_WITH(stage1-systemd-version,
+        AS_HELP_STRING([--with-stage1-systemd-version=version],
+                [systemd version to build]),
+        [RKT_STAGE1_SYSTEMD_VER="$withval"],
+        [RKT_STAGE1_SYSTEMD_VER="v219"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_VER)
+
+## STAGE1: linker-defined custom STAGE1 image path, default is unset
+AC_ARG_WITH(stage1-image-path,
+        AS_HELP_STRING([--with-stage1-image-path],
+                [custom stage1 image path]),
+        [RKT_STAGE1_IMAGE="$withval"],
+        [RKT_STAGE1_IMAGE=""])
+
+RKT_STAGE1_IMAGE_FLAGS=
+# if stage1 image variable is set, add a linker flag to rkt defining the variable
+if test x"${RKT_STAGE1_IMAGE}" != x ; then
+   RKT_STAGE1_IMAGE_FLAGS="-ldflags \"-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'\""
+fi
+
+AC_SUBST(RKT_STAGE1_IMAGE_FLAGS)
+
+## STAGE1 D-Bus parameters
+AC_ARG_WITH(stage1-dbus,
+        AS_HELP_STRING([--with-stage1-dbus=no],
+                [turn of d-bus or not, 'yes' turns it on, 'no' turns it off]),
+        [RKT_STAGE1_DBUS="$withval"],
+        [RKT_STAGE1_DBUS="no"])
+
+AC_SUBST(RKT_STAGE1_DBUS)
+
+AC_ARG_WITH(stage1-dbus-src,
+        AS_HELP_STRING([--with-stage1-dbus-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_DBUS_SRC="$withval"],
+        [RKT_STAGE1_DBUS_SRC="git://anongit.freedesktop.org/dbus/dbus"])
+ 
+AC_SUBST(RKT_STAGE1_DBUS_SRC)
+
+AC_ARG_WITH(stage1-dbus-version,
+        AS_HELP_STRING([--with-stage1-dbus-version=version],
+                [d-bus version to build]),
+        [RKT_STAGE1_DBUS_VER="$withval"],
+        [RKT_STAGE1_DBUS_VER="dbus-1.8.16"])
+
+AC_SUBST(RKT_STAGE1_DBUS_VER)
+
+if test x"${RKT_STAGE1_DBUS}" == xyes ; then
+
+AC_MSG_RESULT([stage1 will build D-Bus])
+
+AC_SEARCH_LIBS([_nss_files_gethostbyname_r], [nss_files], [],
```

Comment:
not sure about musl, but we need this particular one to allow D-Bus read the /etc/passwd and /etc/group otherwise it crashed in attempt to find a user "messagebus"